### PR TITLE
win_get_url: Fixed a few issues with using FTP and added tests

### PR DESCRIPTION
**Pull request by jborean93**
##### SUMMARY
This PR slightly modified win_get_url to fully support FTP as a URL option. This also adds tests to make sure this doesn't regress in the future.

Further changes based on https://github.com/ansible/ansible/pull/39317

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
win_get_url

##### ANSIBLE VERSION
```
devel
```

### CODE REVIEW

**Comment by jhawkesworth** [comment]
Looks good to me, I am +1 for merging this.

File: lib/ansible/modules/windows/win_get_url.py [hunk 1]
```diff
@@ -15,9 +15,10 @@
 ---
 module: win_get_url
 version_added: "1.7"
-short_description: Fetches a file from a given URL
+short_description: Downloads file form HTTP, HTTPS, or FTP to node
```
Comment (nitzmahone) [inline lib/ansible/modules/windows/win_get_url.py]: s/form/from/

**Review by nitzmahone — APPROVED** [review]
Just the one little docs typo- feel free to cherry-pick for 2.5.3 if you like as well

**Comment by jborean93** [comment]
Backport PR to stable-2.5 https://github.com/ansible/ansible/pull/39705

**Pull request merged**